### ISSUE TITLE
[Bug]: Right click to move camera doesn't work if clicking a nameplate

### ISSUE THREAD

**Issue by Sundava** [DelvUI/DelvUI#1159]
### What are you trying to do?

Trying to move the camera using the right click, but the click sometime happens on a health bar nameplate

### What is the expected behavior?

The nameplate should ignore interraction with right click, allowing me to move the camera anyway

### What actually happened?

The right click input is ignored altogether.

### Suggested solution

It see a recent commit talks about fixing an interaction with left click and the health bar nameplate, I would assume something along the line can be applied here.

Otherwise, an sub-option to the `Disable Interraction` setting only for the right-click would fix the issue.

### FFXIV Update

- [X] I have confirmed that I have the latest version of XIVLauncher and DelvUI.

**Comment by Sundava** [comment]
Looking into it a bit more it seems the same happens whn trying to move the camera with left click.
This seems fine because left clicking a nameplate has a function, but this is not the case for the right click

**Comment by Tischel** [comment]
Not possible, sadly.

**Comment by Sundava** [comment]
Hello there, sorry for the necro but after the recent Dalamud API update I fiddled a bit more with my fork of DelvUI, give there were some changes.

Basically, removing the "eating the input" logic from `InputsHelper.WndProcDetour` allows me to use right click to move the camera even if the right click lands on a nameplate, but only if I set `InputsProxyEnabled` (introduced in ac0854ed165d0872204a1904e25c85f54903b24f)  to true via command line. 

However, enabling it means we can't left click on a nameplate to target something anymore.

With `/delvui mouse off`, it's the opposite, left click to target works but we can't move the camera if the left/right click happens to land on a nameplate.

What's strange is that on the initial state after plugin load, the `left click to target` works (albeit after a time, it takes about 2s after plugin load for the left click event on a nameplate to actually do something), AND right clicking on a nameplate does not prevent moving the camera. This behavior disappears after a set of `/delvui mouse off`/`/delvui mouse on`.

I'm not sure if that piques your interest, but it does seem to suggest that there actually is a way to implement this functionality at seemingly no cost ?

The relevant commit on my fork is [this one](https://github.com/Sundava/DelvUI/commit/bc6b46a7f6d5ed3e658d5d7d368e00d3e28685f6), it's currently up to date with only this commit ahead of upstream, if you ever want to take another stab at this. 
From what I can see, it looks like my fork could already be a PR and everything would work, but I'm reticent to create a PR removing some code I haven't written on a project I'm not familiar with
